packages/styles/scss/patterns/sub-patterns/content-item/_content-item.scss

```diff
@@ -20,10 +20,12 @@
   }
   .#{$prefix}--content-item {
     margin-top: $carbon--spacing-07;
-    margin-bottom: $carbon--spacing-07;
+    margin-bottom: $carbon--spacing-09;
```

Comment:
@andysherman2121 just checking, is this specifically for callout with media only or for anything using ContentItem? Seems like this affects a number of components if you check the percy test.

Reply:
I assumed since Callout-with-media is using content-block-simple, that it's a spacing issue with all the components using content item. I'll look into it though to double check